packages/ui-components/src/utils/responsive-utils/use-breakpoint.ts

```diff
@@ -0,0 +1,23 @@
+import { useDimensions } from '@webb-dapp/react-environment/layout';
+import { useMemo } from 'react';
+
+import { size } from './break-points.util';
+
+export default () => {
+  const { width } = useDimensions();
+
+  const isXsOrAbove = useMemo(() => width >= size.xs, [width]);
+  const isSmOrAbove = useMemo(() => width >= size.sm, [width]);
+  const isMdOrAbove = useMemo(() => width >= size.md, [width]);
+  const isLgOrAbove = useMemo(() => width >= size.lg, [width]);
+  const isXlOrAbove = useMemo(() => width >= size.xl, [width]);
+
+  return {
+    breakpoints: size,
+    isXsOrAbove,
+    isSmOrAbove,
+    isMdOrAbove,
+    isLgOrAbove,
+    isXlOrAbove,
+  };
+};
```

Comment:
instead of multiple memoized values return a single object

`const breakPoints  = useMemo(() =>{
return  {
breakpoints: size,
isXsOrAbove :width >= size.xs,
isSmOrAbove :width >= size.sm,
isMdOrAbove :width >= size.md,
isLgOrAbove :width >= size.lg,
isXlOrAbove :width >= size.xl,
}
  }
} , [width])`